### DISCUSSION TITLE
DirSync: Support compare by content with archives

### DISCUSSION THREAD

**Discussion by ltguillaume** [doublecmd/doublecmd#1111] [Suggestions]
### Description

Personally, this is the thing I use most of all the features in Total Commander, so I sorely miss it in DC.

When "by content" is checked in TC and _one_ of the panes is a (self-extracting) archive, TC extracts and creates checksums for the files from the archive and compares them to the files from the other pane.

**Comment by lars18th** [comment]
This is similar to #1584
